library/src/main/java/io/ona/kujaku/wmts/model/common/ows/LanguageStringType.java

```diff
@@ -0,0 +1,26 @@
+package io.ona.kujaku.wmts.model.common.ows;
+
+import org.simpleframework.xml.Attribute;
+import org.simpleframework.xml.Text;
+
+/**
+ * Describe a ows:LanguageStringType element
```

Comment:
Cool work, the WMTS specification which could not render due to missing elements now works 👍 

Fix the typo `Describe` to `Describes` in the doc of this class and the other classes